update/Pos/Update/Poll/Logic/Base.hs

```diff
@@ -81,12 +81,12 @@ confirmBlockVersion confirmedEpoch bv =
 -- Specifically, one of the following conditions must be true.
 -- • Given block version is equal to last adopted block version.
 -- • Given block version is competing
-canCreateBlockBV :: MonadPollRead m => BlockVersion -> m Bool
-canCreateBlockBV bv = do
-    lastAdopted <- getAdoptedBV
+canCreateBlockBV :: MonadPollRead m => BlockVersion -> BlockVersion -> m Bool
```

Comment:
It would be good to introduce newtype wrappers or use `Tagged` here to make it harder to confuse the order of `BlockVersion`s, but it's minor.